tmt/cli.py

```diff
@@ -338,9 +366,10 @@ def run_tests(context, **kwargs):
     callback = run.resultcallback
 
 
-@callback()
+# TODO: commands is unknown, needs revisit
+@callback()  # type: ignore
```

Comment:
@happz any idea about this one? I am unable to convince mypy about the type of this decorator:

```
tmt/cli.py:369: error: Untyped decorator makes function "finito" untyped
```

Reply:
Probably not much we can do about it, we'd have to annotate the decorator... I'd add `ignore` and move on.

Reply:
thanks